### DISCUSSION TITLE
CUE Community Call 2023-10-03

### DISCUSSION THREAD

**Discussion by myitcv** [cue-lang/cue#2613] [Community Calls]
We are excited to confirm Tuesday, **3 October, 2023 at 1430 UTC** for our next CUE Community Call.

Please reply with any questions/topics you would like to see covered.

### Agenda

- Releases and language features updates (v0.6.0 and v0.7.0)
- Unity update + information on unity onboarding
- WebAssembly update
- [Modules proposal updates](https://github.com/cue-lang/cue/discussions/2330) (see also the [feedback sessions](https://github.com/cue-lang/cue/discussions/categories/announcements?discussions_q=is%3Aopen+category%3AAnnouncements+modules+feedback+session))
- [alpha.cuelang.org](http://alpha.cuelang.org/) update

### Meeting details

https://meet.google.com/uka-abmk-svy

This session will also be recorded for those unable to join us live.

### CUE Community shared calendar

[Subscribe to the shared calendar](https://calendar.google.com/calendar/u/0?cid=Y19lNzkxMWQ5OWQ4ZGIyMmU2ZTVjMzhkMTVkNjY2ZTVlNjdiNWE5ODNkZWU4N2JmNTU2NDY3NzI1OGIxYjJhMTFhQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20).  

### Recording

https://youtu.be/-pW5xfrdoqo

### Slides

https://docs.google.com/presentation/d/1TRmIwsxZon_5DxNWl6q3qZYxSBz1S2PLPkXwxCfc164/edit?usp=sharing

### Auto-generated transcript

https://docs.google.com/document/d/1NTbu5wO4HWwPRTI3lVf1u9RW3kX_JiZ0cr9MXINX3MU/edit?usp=sharing

**Comment by myitcv** [comment]
Per @talentedmrjones on Slack

> How is 0.7 looking? Performance has some of my tools stuck on 0.4.3 and I'm eager to move on and be up to date

**Comment by dominikdm** [comment]
Updated with links to the recording, slides and a transcript. Thanks for the good discussion yesterday!
